Support torch dataloader without torch formatting (#5357)
huggingface/datasets: docs/source/use_with_pytorch.mdx

### Stream data

> Loading a dataset in streaming mode is useful to progressively download the data you need while iterating over the dataset.

Reviewer: ```suggestion
Loading a dataset in streaming mode allows one to iterate over the dataset without downloading it on disk.
```
incorrect streaming description from past again 👻

There is the same in line 152 (in Optimize data loading section) but I can't fix it there.
